### DISCUSSION TITLE
Color of Component  <InputNumber />

### DISCUSSION THREAD

**Discussion by sy0uta1k0n** [ant-design/ant-design#41158] [Ideas]
When we set InputNumber as disable, its text color is differenrt with other inputs'. 

I know we can change its css to keep same style, but any plan to modify it in official?

<img width="166" alt="image" src="https://user-images.githubusercontent.com/13308844/224215770-f9f1f17b-1bbf-40c2-afd3-0bbae17d386d.png">
<img width="453" alt="image" src="https://user-images.githubusercontent.com/13308844/224216231-18a09c06-5367-42f1-bdc3-70c5cf5935ee.png">

**Comment by MadCcc** [comment]
Maybe bug here. Feel free to open issue with reproduction.
